test/e2e/common/container_probe.go

```diff
@@ -63,7 +63,7 @@ var _ = framework.KubeDescribe("Probing container", func() {
 	framework.ConformanceIt("with readiness probe should not be ready before initial delay and never restart [NodeConformance]", func() {
 		containerName := "test-webserver"
 		p := podClient.Create(testWebServerPodSpec(probe.withInitialDelay().build(), nil, containerName, 80))
-		f.WaitForPodReady(p.Name)
+		framework.ExpectNoError(e2epod.WaitTimeoutForPodReadyInNamespace(f.ClientSet, p.Name, f.Namespace.Name, framework.PodStartTimeout))
```

Comment:
Did we intend to add the `ExpectNoError` here?

Reply:
> Thanks for all your work to clean up these tests.
> 
> I'm having trouble seeing where this code actually removes a dependency - very possible I'm missing something... or perhaps its a precursor to a diff that will remove some imports?

see https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/framework.go#L54  and

Reply:
> Did we intend to add the `ExpectNoError` here?

This may not be related to this pr, but the BB function will return an error, and you see other similar code that calls this BB function to handle errors. I propose to add error handling here. What do you think?

Reply:
I know this kind thing depends on each person.
But I prefer to see a single PR should do a single thing, in this case this PR should just replace functions without doing anything because that makes unnecessary discussion like this.
It is better to add error handling with another PR if necessary.

Reply:
@alejandrox1  Thank you for your comment, as @oomichi  said, this pr is currently for the purpose of eliminating dependencies, I can handle this error  in another pr. 😄